DataFormats/L1TCalorimeterPhase2/src/classes_def.xml

```diff
@@ -42,5 +42,13 @@
   </class>
   <class name="l1tp2::DigitizedClusterGTCollection" />
   <class name="edm::Wrapper<l1tp2::DigitizedClusterGTCollection>" /> 
+
+  <class name="l1tp2::CaloPFCluster"  ClassVersion="4">
+         <version ClassVersion="4" checksum="4176166177"/>
+  </class>
+  <class name="std::vector<l1tp2::CaloPFCluster>" />
+  <class name="l1tp2::CaloPFClusterCollection" />
+  <class name="edm::Wrapper<l1tp2::CaloPFClusterCollection>" />
```

Comment:
They will want this class version to start from 3.

Reply:
Changed to 3